keras/backend/tensorflow_backend.py

```diff
@@ -4220,13 +4241,16 @@ def range_less_than(_, current_input):
     label_ind = tf.boolean_mask(label_array, dense_mask)
 
     batch_array = tf.transpose(tf.reshape(tf.tile(tf.range(label_shape[0]),
-                                                  max_num_labels_tns), reverse(label_shape, 0)))
+                                                  max_num_labels_tns),
+                                          reverse(label_shape, 0)))
```

Comment:
Please shorten the line by using temporary variables, it will be easier to read.